WordPress/Classes/ViewRelated/Reader/Tab Navigation/ReaderTabView.swift

```diff
@@ -13,6 +13,7 @@ class ReaderTabView: UIView {
 
     private let viewModel: ReaderTabViewModel
 
+    private var filteredTabs: [(index: Int, topic: ReaderAbstractTopic)] = []
```

Comment:
I'm wondering if there's ever a situation where the topic entity could be deleted, while a reference to it is still stored here.  I tried to test this in the app by following/unfollowing a site and by login out and back in, both while the filter was set.  There were no issues as far as I could tell (yay!) but this is till a question nagging me. 

For paranoia sake, would it be safer to store the objectID rather than the entity or am I worrying too much?

Reply:
I can't think of reason to do so. Since you stated the cases you thought of, the only other one I came up with was deleting a site. So I filtered by a site then deleted the site. The filter did remain and showed content, so there were no adverse effects. The site did disappear from the site filter list, so after I cleared the filter I wasn't able to see the site anymore. As far as I can tell it works fine as-is.

Tangentially, one thing I did notice is that if you filter by a site or tag, then go to Manage on the nav bar and unfollow the site/tag, the filter is not cleared. I'll see what I can do about that.

Reply:
Nice catch!